app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,230 @@
+    "description": "API product for classic pizzas",
+    "portal_ids": "PORTAL_ID",
+    "labels": {
+        "env": "test"
+    }
+  }'
+```
+1. Create the `v1` version for the `Classic Pizzas` API product and publish it:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products/<api-product-id>/product-versions \
+  --data '{
+    "id": "GATEWAY_SERVICE_ID",
+    "name": "v1",
+    "gateway_service": {
+        "runtime_group_id": "RUNTIME_GROUP_ID"
+    },
+    "publish_status": "published",
+  }'
+```
+{% endnavtab %}
+{% endnavtabs %}
+
+### Create developer teams
+
+Next, let's create two developer teams: "Delivery Partners" with API Viewer and Consumer permissions and "New Partners" with API Consumer permissions.
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon dev-portal %} **Dev Portal**.
+1. Click **Teams** in the sidebar and then click **Add team**.
+1. Enter the following:
+    * **Team name:** Delivery Partners
+    * **Description:** Team of vetted delivery partners that can view and consume APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Classic Pizzas** product from the dropdown menu and select **API Consumer** and **API Viewer** from the **Add roles** dropdown menu.
+1. Click **Teams** in the sidebar and click **New Team** again and enter the following:
+    * **Team name:** New Partners
+    * **Description:** Team of unvetted partners that can only view APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Classic Pizzas** product from the dropdown menu and select **API Viewer** from the **Add roles** dropdown menu. 
+{% endnavtab %}
+{% navtab API %}
+{:.note}
+> **Note:** Make sure you replace the placeholders in the following API requests with information specific to your environment. 
+
+1. Create the `Delivery Partners` team:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/<portal-id>/teams \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "name": "Delivery Partners",
+    "description": "Team of vetted delivery partners that can view and consume APIs"
+  }'
+```
+1. Create the `New Partners` team:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/<portal-id>/teams \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "name": "New Partners",
+    "description": "Team of unvetted partners that can only view APIs"
+  }'
+```
+1. You can verify that the teams have been created by making a GET request to the teams endpoint:
```

Comment:
This step would be more helpful if you included the response and the relevant parts of the response that verify if a team was created.

Reply:
You can remove this step though and link out to the API spec endpoint.